test/generic.spec.js

```diff
@@ -0,0 +1,174 @@
+var fs = require('fs');
+var Promise = require('bluebird');
+var readFile = Promise.promisify(require('fs').readFile);
+var expect = require('chai').expect;
+var async = require('async');
+var Converter = require('csvtojson').Converter;
```

Comment:
please post your log and npm version. At npm 3.5.3 all installs with no problems